docs/data/data-grid/filtering/CustomSelectionOperator.tsx

```diff
@@ -62,9 +60,9 @@ export default function CustomSelectionOperator() {
      * Function that takes an operator and wrap it to skip filtering for selected rows.
      */
     const wrapOperator = (operator: GridFilterOperator) => {
-      const getApplyFilterFn = (
-        filterItem: GridFilterItem,
-        column: GridStateColDef,
+      const getApplyFilterFn: GridFilterOperator['getApplyFilterFn'] = (
```

Comment:
It's actually better to type the whole function rather than arguments. And this way we can avoid exporting `GridStateColDef`